cmd/geth/main.go

```diff
@@ -71,6 +71,8 @@ var (
 		utils.NoUSBFlag,
 		utils.DirectBroadcastFlag,
 		utils.DisableSnapProtocolFlag,
+		utils.DisableDiffProtocolFlag,
```

Comment:
I mean it is not proper to introduce such low level flag. Especially it is related with other flags. 
For example: when `TriesVerifyModeFlag` need trust protocol while `EnableTrustProtocolFlag` is false, it is become complicated.

Reply:
Consider we should disable diff protocol in verify node, and enable trust protocol, we didn't have better solution temporary.